document/js-api/index.bs

```diff
@@ -561,6 +568,8 @@ enum IndexType {
   "i64",
 };
 
+typedef ([EnforceRange] unsigned long long or bigint) IndexValue;
```

Comment:
Do we need to file a bug as mentioned in the warning here: https://webidl.spec.whatwg.org/#dfn-union-type ?

Reply:
It would probably be wise, yes. Hopefully it's a quick rubber stamp and we can move on.

I'm traveling at the moment, so I would appreciate if you could open that issue.

Reply:
https://github.com/whatwg/webidl/issues/1426